### ISSUE TITLE
[regression] `isConnected` with Dymo Connect

### ISSUE THREAD

**Issue by Roman991** [dymosoftware/dymo-connect-framework#36]
With old dymo labelwriter the response of `getPrinters()` was correct.

With new Dymo connect (actually 1.4.2.82), printers are always offline.

idk if its more related to this framework of to a wrong dymo connect response.
Tried on different pcs (both win10 and 11)

```javascript
getPrinters() response ===>>
<Printers>
  <LabelWriterPrinter>
    <Name>DYMO LabelWriter 450 Turbo</Name>
    <ModelName>DYMO LabelWriter 450 Turbo</ModelName>
    <IsConnected>False</IsConnected>
    <IsLocal>True</IsLocal>
    <IsTwinTurbo>False</IsTwinTurbo>
  </LabelWriterPrinter>
</Printers>```

**Comment by grezniczek** [comment]
I am also seeing this. getPrinters reports `<IsConnected>False<//IsConnected>`, but printing through the JS framework works nonetheless.

Windows 10, Dymo Connect 1.4.2.82

**Comment by drewjoh** [comment]
Just discovered this also.  I can ignore the status and print regardless and it works, as long as the printer is connected.

The **big issue** is:  if the printer is **not** connected, labels queue up and get printed once the printer is connected again!

A likely scenario:  a user who clicks to print, gets no print, clicks again... and again, and again... then figures out it's not connected... connects it, and now gets all those clicks as prints.  😕 

Also, it makes my nice status indicator not work on Windows (I've been implementing and testing on Mac).  😞 

<img width="238" alt="CleanShot 2022-01-14 at 16 49 23@2x" src="https://user-images.githubusercontent.com/66233/149595556-acc454e6-131d-4596-9a89-2c96feddabd0.png">

Tested on:  Windows 10, Dymo Connect 1.4.2.82

**Comment by Longhorn74** [comment]
We are experiencing the same which is preventing our customers from using the LabelWriter 550 model with our web application.  Our customers want to purchase the newer model, especially since the 450 is backordered, but cannot because of this issue.

@dymosoftware, please advise.  This is a critical issue for us and we can't get any assistance from Dymo Customer Support.  We've spent 6+ hours waiting on hold only to be told they can't help in any way and hang up on us.

**Comment by dymosoftware** [comment]
We have beta version for 1.4.3 in https://github.com/dymosoftware/Beta-Version.
We are going to release 1.4.3 soon, but IsConnected issue and 550 printer issue is not able to be found in the Beta-Version.
